app/views/dashboard/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Dashboard#index</h1>
-<p>Find me in app/views/dashboard/index.html.erb</p>
+
+<nav class="navbar navbar-dark bg-primary" style="background-color: #e3f2fd;">
+  <a class="navbar-brand" href="#"><h1>Colony W</h1></a>
+
+  <div class="dropdown">
+  <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    User Account
+  </button>
+  <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+    <a class="dropdown-item" href="/users">My Profile</a>
```

Comment:
We should use some `link_to` instead of hardcoded links here but I guess we can handle it later :D

Reply:
I can fix that sooner.
P/s: Sorry I didn't check the comments.